hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -659,15 +660,6 @@ public OmKeyInfo lookupKey(OmKeyArgs args, String clientAddress)
           });
         }
       }
-      // Refresh container pipeline info from SCM
```

Comment:
Can we also move the generation of the secret token also outside of the lock?

Reply:
Thanks for the review. Sure, will do it.